extraction/video_extractor.py

```diff
@@ -0,0 +1,43 @@
+"""This script checks whether DOM has video tag or not and
+creates and returns the Video object"""
+
+import re
+
+from extraction.data_models.video import Video
+
+VIDEO_EXTENSIONS_PATTERN = re.compile(r'^.*\.(mp4|ogg|mov|webm)')
+
+
+def get_size(node):
```

Comment:
Move private functions down. Also, add _ to it.

Reply:
moved to extraction_utils